docs/cheatsheet/cli.rst

```diff
@@ -0,0 +1,85 @@
+.. _ref_cheatsheet_cli:
+
+CLI
+===
+
+Create a database:
+
+.. code-block:: bash
+
+    $ edgedb --admin create-database my_new_project
```

Comment:
I think this is the old way of doing this, no? @elprans

Reply:
Yes. `--admin` is deprecated, all examples should use `-I<instance-name>` instead.  Also, we should be recommending creating instances instead of databases for new projects.

Reply:
I've installed edgedb on my Ubuntu laptop from scratch, re-tried the commands and updated the cheat sheet accordingly.